src/parameters/update_parameters.jl

```diff
@@ -40,6 +40,11 @@ function update_parameter_values!(
     horizon = get_time_steps(get_optimization_container(model))[end]
     components = get_available_components(V, get_system(model))
     for component in components
+        name = PSY.get_name(component)
+        @debug "Updating parameter value with time series" name,
```

Comment:
there is a recorder entry for this. It is better to avoid @debug statements in hot loops like this. Alternatively, we can just add a comment to uncomment the line for debugging.